cmd/cli/operations/update_job_test.go

```diff
@@ -335,11 +368,13 @@ func TestUpdateJobShouldReturnAnErrorWhenMultipleRunningJobsAreFound(t *testing.
 		},
 	}
 
+	// flink-deployer wont want to update (stop/start) an undesired job
+	// when there are two running jobs with same name. So it must abort the update
 	err := operator.Update(UpdateJob{
-		JobNameBase:   "WordCountStateful",
+		JobNameBase:   "WordCountStateful v1.0",
```

Comment:
Don't we want to use `WordCountStateful` as the base name here to keep things consistent? For the starts with logic, it shouldn't matter if 2 V1.0 or 1 V1.0 and V1.1 is running.

Reply:
Fixed it. 

As a side conversation, 
For my usecase, I need a slight change in behaviour to the update operation. I want the update to continue deploying if there are no instances running. I am ansible automating our deployment and I've done that change in my local fork and my automation seems to work well. Do you think I should create a PR into your repo?

Reply:
When we rewrote the deployer to use the REST API we actually decided not to have this behaviour anymore. For deploying, we use the `deploy` task. However, maybe we can add it again, but not as default behaviour but by adding a new flag. Something like `--fallback-to-deploy`? @nielsdenissen what do you think?

@joshuavijay can you open a separate issue to discuss this?

Reply:
I can imagine this is a common use case and agree that we can provide this functionality (saves a user from implementing it him/herself). Definitely shouldn't be the default indeed, as that could be very confusing.